java/core/src/main/java/sleeper/core/util/CoreUtils.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2022-2023 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sleeper.core.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CoreUtils {
+    private CoreUtils() {
+    }
+
+    private static final long K_COUNT = 1_000;
+    private static final long M_COUNT = 1_000_000;
+    private static final long G_COUNT = 1_000_000_000;
+    private static final long T_COUNT = 1_000_000_000_000L;
+
+    public static String formatBytes(long fileSize) {
```

Comment:
Can we have a version of this that includes both the real count and the human-readable version? This is used a bit inconsistently right now.